docs/ipr/index.md

```diff
@@ -1,5 +1,5 @@
 
-# About
+# About IPR
 
 Integrated Pipeline Reports (IPR) is the main reporting application of IPR. It is a web application
```

Comment:
- there's a phrase in this file 'across 3 repositories' - maybe change to 'across three repositories'
- maybe change 'These are either create' to 'These are either created', in the third paragraph
- maybe change 'manually curated post-report creation' to 'manually curated after report creation'
- either change the semicolon in the last sentence to a comma or replace the first comma in this sentence with a semicolon